packages/docs/src/locales/en/en.json

```diff
@@ -661,16 +661,18 @@
         "canBeFocused": "If false, then input can not be focused using mouse or keyboard."
       },
       "events": {
```

Comment:
![image](https://user-images.githubusercontent.com/5394573/147936995-99a59893-b78a-482f-ae8a-9288a53b8705.png)

here, if you tab away from this component - address bar is getting selected.
Instead we should select next time input.
Could be done by handling events on input instead of dropdown.

Reply:
Not resolved yet, we can also move this to some other issue if too hard to fix.

Reply:
Fixed